app/components/operator-form/country-region-select/template.hbs

```diff
@@ -1,16 +1,21 @@
 <div class="col-sm-4 col-xs-12 input-section">
-  <div class="input-form-label">Country</div>
-  <div class="input-form-field">
-    {{#power-select options=countries searchField="name" selected=selectedCountry onchange=(action "setCountry") as |country|}}
-      {{country.name}}
-    {{/power-select}}
-  </div>
+	<div class="input-form-label">Country</div>
+	<div class="input-form-field">
+		{{#power-select options=countries searchField="name" selected=selectedCountry onchange=(action "setCountry") as |country|}}
+			{{country.name}}
+		{{/power-select}}
+	</div>
 </div>
 <div class="col-sm-4 col-xs-12 input-section">
-  <div class="input-form-label">State</div>
-  <div class="input-form-field">
-    {{#power-select options=selectedRegions searchField="name" selected=selectedRegion onchange=(action "setRegion") as |region|}}
-      {{region.name}}
-    {{/power-select}}
-  </div>
+	<div class="input-form-label">State</div>
+	<div class="input-form-field">
+		{{#power-select options=selectedRegions searchField="name" selected=selectedRegion onchange=(action "setRegion") as |region|}}
+			{{region.name}}
+		{{/power-select}}
+	</div>
 </div>
+<!-- Municipality/Region moved to here -->
+<div class="col-sm-4 col-xs-12 input-section">
```

Comment:
I moved Municipality/Region into `country-region-select` component. Ember put its own wrapper on top of its component, and this blocks some style if you are using `:last-child` selector. (which is used to style table right now) *Please re-edit this if this affects any functionality! *